pep-0008.txt

```diff
@@ -690,16 +690,15 @@ Comments that contradict the code are worse than no comments.  Always
 make a priority of keeping the comments up-to-date when the code
 changes!
 
-Comments should be complete sentences.  If a comment is a phrase or
-sentence, its first word should be capitalized, unless it is an
-identifier that begins with a lower case letter (never alter the case
-of identifiers!).
+Comments should be complete sentences.  The first word should be 
+capitalized, unless it is an identifier that begins with a lower case
+letter (never alter the case of identifiers!).
 
-If a comment is short, the period at the end can be omitted.  Block
-comments generally consist of one or more paragraphs built out of
-complete sentences, and each sentence should end in a period.
+Block comments generally consist of one or more paragraphs built out of
+complete sentences, with each sentence ending in a period.
 
-You should use two spaces after a sentence-ending period.
+You should use two spaces after a sentence-ending period in multi
+sentence comments, except after the final sentence.
```

Comment:
Explicitly ask for double space separated sentences, without suggesting trailing whitespace.